cdap-authorization-dataset-plugin/src/main/java/co/cask/cdap/security/authorization/DatasetBasedAuthorizer.java

```diff
@@ -67,11 +71,22 @@ public ACLDataset get() {
         return new ACLDataset(table);
       }
     };
+    Properties properties = context.getExtensionProperties();
+    if (properties.containsKey("superusers")) {
```

Comment:
For the extensions that we have implemented so far, yes, they need at least one super user. Do you think we should log a warning or something if it is not configured with at least one superuser?